gems/pending/disk/modules/MSVSDiffDisk.rb

```diff
@@ -50,7 +50,6 @@ def d_init
       #
       # TODO: need details on Mac Alias Blob.
       #
-      # getParentPathMac(locator)
```

Comment:
What about the MacX case with a comment from 03/14/2007? (line 57/56 below)
Seems like we'll never do this and we might want to remove these two branches of the case statement.  I can help resurrect this in git if needed.